CodeMaid/UI/Converters/NameParametersToTextBlockConverter.cs

```diff
@@ -89,46 +184,46 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
                         }
                         else
                         {
-                            textBlock.Inlines.Add(CreateRun(", "));
+                            textBlock.Inlines.Add(CreateItalicRun(", "));
                         }
 
                         try
                         {
                             textBlock.Inlines.Add(CreateTypeRun(TypeFormatHelper.Format(param.Type.AsString) + " "));
-                            textBlock.Inlines.Add(CreateRun(param.Name));
+                            textBlock.Inlines.Add(CreateItalicRun(param.Name));
                         }
                         catch (Exception)
                         {
-                            textBlock.Inlines.Add(CreateRun("?"));
+                            textBlock.Inlines.Add(CreateItalicRun("?"));
                         }
                     }
                 }
                 catch (Exception)
                 {
-                    textBlock.Inlines.Add(CreateRun("?"));
+                    textBlock.Inlines.Add(CreateItalicRun("?"));
                 }
 
-                var closer = GetClosingString(codeItem);
+                var closer = GetClosingString(codeItemWithParams);
                 if (closer != null)
                 {
-                    textBlock.Inlines.Add(CreateRun(closer));
+                    textBlock.Inlines.Add(CreateItalicRun(closer));
                 }
             }
 
             return textBlock;
         }
 
-        /// <summary>
-        /// Converts a value.
-        /// </summary>
-        /// <param name="value">The value that is produced by the binding target.</param>
-        /// <param name="targetType">The type to convert to.</param>
-        /// <param name="parameter">The converter parameter to use.</param>
-        /// <param name="culture">The culture to use in the converter.</param>
-        /// <returns>A converted value. If the method returns null, the valid null value is used.</returns>
-        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        ///// <summary>
+        ///// Converts a value.
+        ///// </summary>
+        ///// <param name="value">The value that is produced by the binding target.</param>
+        ///// <param name="targetType">The type to convert to.</param>
+        ///// <param name="parameter">The converter parameter to use.</param>
+        ///// <param name="culture">The culture to use in the converter.</param>
+        ///// <returns>A converted value. If the method returns null, the valid null value is used.</returns>
+        public object[] ConvertBack(object value, Type[] targetTypes, object parameter, CultureInfo culture)
         {
-            throw new NotImplementedException();
+            return new[] {Binding.DoNothing, Binding.DoNothing};

```

Comment:
Is there an advantage to the DoNothing's over NotImplemented?  Does this code get hit?